### DISCUSSION TITLE
Moving go files, gowebly run stops working

### DISCUSSION THREAD

**Discussion by frankie-mur** [gowebly/gowebly#43] [Q&A]
May be a noob question, but I want to start improving the package structure of my project 
ie. move the go files to a cmd/web directory. Once I did this the `gowebly run` command stops working (I believe is looking to run the go files from the . directory) 
<img width="898" alt="image" src="https://github.com/gowebly/gowebly/assets/34355192/0c81b3ab-48ef-4f4b-b523-54aac7f87287">
I know it has to do with air, I tried adding a custom .air.toml file but still not getting it right. Can someone please provide some pointers here?

**Comment by koddr** [comment]
Hi, @frankie-mur 

Not really sure what you want the Gowebly CLI to do for you. 

I can see from the screenshot that you are trying to run your project that doesn't have the required files (CLI says `no go files in /Users/…`). But this has nothing to do with how the Gowebly CLI works.
